uwsdora.module

```diff
@@ -106,3 +106,52 @@ function uwsdora_views_api() {
     'path' => drupal_get_path('module', 'uwsdora') . '/views',
   );
 }
+
+/**
+ * Implements hook_convert_mods_to_citeproc_jsons_alter().
+ */
+function uwsdora_convert_mods_to_citeproc_jsons_alter(&$output, $mods) {
+  module_load_include('inc', 'citeproc', 'includes/converter');
+
+  $pages = $mods->xpath("//mods:mods[1]/mods:relatedItem[@type='host']/mods:part/mods:extent[@unit='pages']");
+
+  if (empty($pages)) {
+    $pages = $mods->xpath("//mods:mods[1]/mods:relatedItem[@type='host']/mods:part/mods:extent[@unit='page']");
```

Comment:
Probably not worth pursuing, but this _could_ be done in the first XPath; however, could get kind of ugly... Would be something like:
```php
$pages = $mods->xpath("//mods:mods[1]/mods:relatedItem[@type='host']/mods:part/mods:extent[@unit='pages' or (@unit='page' and not(../mods:extent[@unit='pages']))]");
```
... Shrug?

... Actually, digging a little deeper, the motivation of this alter is just to account for a difference in the hierarchy of elements... may be a good idea to point at the [upstream/core implementation](https://github.com/Islandora/islandora_scholar/blob/7.x/modules/citeproc/includes/converter.inc#L292-L339) and say "We're doing that", kind of thing, but with the `relatedItem[@type="host"]` kind of thing... Kind of highlighting the copypasta nature.

... Could even wrap up the page handling business into a(n anonymous) function, to make things more explicit? Shrug?